### ISSUE TITLE
[Improvement] Switch to Sweetalert2

### ISSUE THREAD

**Issue by kiddtang** [Laravel-Backpack/ideas#69]
Currently Backpack using https://github.com/t4t5/sweetalert

However, Sweetalert2 API & Documentation are way more powerful and complete.
https://sweetalert2.github.io/

**Comment by tabacitu** [comment]
Yup, I agree, we should upgrade to sweetalert v2. It'll be a breaking change, though, because developers might have called it themselves throughout the admin panel. So we can only do it inside the 4.2 launch. Tagged appropriately so we don't forget about it.

Thanks @kiddtang .
